.github/workflows/test.yaml

```diff
@@ -165,8 +165,12 @@ jobs:
         run: source /root/venv/bin/activate && echo "PATH=$PATH" >> $GITHUB_ENV
       - name: Make Log Dir
         run: mkdir -p /tmp/weave/log
-      - name: Copy over built assets
-        run: cp -r /root/weave-js-build/* ./weave/frontend
+      - name: Build assets
+        run:
+          cp -r /root/weave-js-build/* ./weave/frontend && cd ./weave/frontend \
+          yarn build && \
+          echo $SHA1 > ./build/sha1.txt && \
+          rm -rf node_modules
```

Comment:
@jlzhao27 has a new action step that builds the frontend and uploads a tarbal to a bucket.  We could either make all flows that need the frontend depend on that step or atleast make this check if it already has a build and only re-build if it doesn't exist.

Reply:
Ya, you should be able to check SHA in the `index.html` or the sha1.txt to find the bundle url. I think you have to parse the index file since that is the sha we use to upload the bundle. Here is an example automated build that should've uploaded assets: https://github.com/wandb/weave/commit/5375458ac2a0762707d893453a5be0d610f74391